x/model/client/cli/tx_model.go

```diff
@@ -43,6 +45,10 @@ func CmdCreateModel() *cobra.Command {
 		Short: "Add new Model",
 		Args:  cobra.ExactArgs(0),
 		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			if commissioningFallbackURL != "" && discoveryCapabilitiesBitmask == 0 {
```

Comment:
What if --discoveryCapabilitiesBitmask is set but the value==0 (--discoveryCapabilitiesBitmask=0)?
Please add a corresponding test. The expectation here that no error should be raised.